Realm/RLMClassInfo.mm

```diff
@@ -38,11 +38,11 @@
 : realm(realm), rlmObjectSchema(rlmObjectSchema), objectSchema(objectSchema) { }
 
 RLMClassInfo::RLMClassInfo(RLMRealm *realm, RLMObjectSchema *rlmObjectSchema,
-                           std::unique_ptr<const realm::ObjectSchema *> objectSchema)
+                           std::unique_ptr<realm::ObjectSchema> schema)
 : realm(realm),
 rlmObjectSchema(rlmObjectSchema),
-objectSchema(*objectSchema),
-dynamicObjectSchema(std::move(objectSchema)) { }
+objectSchema(std::move(&*schema)),
```

Comment:
Newline before comma for member initializers.

The std::move() doesn't do anything here. Moving a pointer and copying it are the exact same operation.